### PR TITLE
documenting default API (dummy)

### PR DESCRIPTION
**Pull request by manzidavidrw**
### what does this PR do
documentation of endpoints

### Description of the task to be completed
should have the following endpoints but this is what we have so far

GET api/dummy/

### how should this be manually tested
after cloning this repo do npm run dev to run 

use browser and visit http://localhost:5000/api/dummy/swagger  to test it locally 
when it get deployed you will get another link here as well 

### Any background context you want to provide?
it run locally so far but whenever other we have all our endpoints we will provide the link for better usage

### What are the relevant pivotal tracker stories
#187300138

### CODE REVIEW

**Review by IRASUBIZA-Elyse — APPROVED** [review]

File: src/config/config.js
Comment (fab-ryan) [inline src/config/config.js:1]: why did you add dialect here we all know it's postgress

File: src/config/config.js [hunk 1]
```diff
@@ -20,6 +21,7 @@ const getDatabaseConfig = () => {
         password: process.env.PROD_DATABASE_PASSWORD ?? "",
         host: process.env.PROD_DATABASE_HOST ?? "",
         port: process.env.PROD_DATABASE_PORT ?? "",
+        dialect: "postgres",
```
Comment (fab-ryan) [inline src/config/config.js]: same as here

File: src/database/db.ts [hunk 2]
```diff
@@ -1,8 +1,13 @@
 import { Dialect, Sequelize } from "sequelize";
 import getDatabaseConfig from "../config/config";
 import Models from "./models";
+import dotenv from "dotenv";
+
+
```
Comment (fab-ryan) [inline src/database/db.ts]: why this space

**Review by fab-ryan — COMMENTED** [review]
You are missing PR description

**Review by fab-ryan — CHANGES_REQUESTED** [review]
Please follow the format

File: src/config/config.js
Comment (TuyishimireEric) [reply src/config/config.js:1]: I think this dialect will be useful when we run `npm run migrate` or `npm run seed`, without this line the connection fails

File: src/config/config.js (hunk 1 above)
Comment (TuyishimireEric) [reply src/config/config.js]: This line also, I think we may supply the dialect  run `npm run migrate` or `npm run seed`,

@fab-ryan, can we get another alternative

File: src/config/config.js
Comment (fab-ryan) [reply src/config/config.js:1]: we need that when we are not sure target of database but all of use are using progress no need to add it but in future you might have to use sqlite or mariadb

Comment (TuyishimireEric) [reply src/config/config.js:1]: Okay, thanks

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @manzidavidrw   👋 👋 

#### I'm @TuyishimireEric,
Good job on setting API documentation using swagger🥇 

done

Your task is complete!  Congratulations! 🎉 

### Cheers and Happy coding!👏👏👏

File: tsconfig.json [hunk 3]
```diff
@@ -107,4 +107,4 @@
     "skipLibCheck": true /* Skip type checking all .d.ts files. */
   },
   "include": ["src/**/*.ts", "src/config/config.js"]
-}
+}
```
Comment (fab-ryan) [inline tsconfig.json]: we need new line here

**Review by fab-ryan — APPROVED** [review]
